npm-shrinkwrap.json

```diff
@@ -1208,8 +1205,7 @@
     "currently-unhandled": {
       "version": "0.4.1",
       "from": "currently-unhandled@>=0.4.1 <0.5.0",
-      "resolved": "https://registry.npmjs.org/currently-unhandled/-/currently-unhandled-0.4.1.tgz",
-      "dev": true
+      "resolved": "https://registry.npmjs.org/currently-unhandled/-/currently-unhandled-0.4.1.tgz"
```

Comment:
Indeed. `pretty-bytes` has no dependencies, so there must be something going on here. Can you double check @Shou ?